run.py

```diff
@@ -41,6 +40,14 @@ def run_step(self, name: str, script_path: str, timeout: int = 10 * 60) -> StepR
         print(stderr)
         print(stdout)
 
+        if not get_durtion_from_stdout:
+            elapsed = time.time()
+            duration = round(elapsed - start, 3)
+        else:
+            try:
+                duration = float(stdout)
+            except ValueError:
+                duration = None
```

Comment:
If there was a value error:
print a warning and then fall back to the normal duration calculation model.